tsconfig.json

```diff
@@ -17,7 +17,8 @@
     "types": ["vite/client", "node"],
     "typeRoots": ["src/types", "node_modules/@types"],
     "paths": {
-      "@/gs/*": ["./src/generated-sources/*"]
+      "@gs": ["./src/generated-sources"],
+      "@gs/*": ["./src/generated-sources/*"]
```

Comment:
do we need both? Isn't the second one enough?

Reply:
Yes, before you ask I also tried:
```
"@gs*": ["./src/generated-sources*"],
"@gs*": ["./src/generated-sources", "./src/generated-sources/*"],
```
but no, TS causes a lot of errors. I asked the Intellij AI also. Only the above config supports imports
```
./generated-sources
../generated-sources
../../generated-sources
.../../generated-sources
```

Reply:
ok